ERCS/eip-draft_title_abbrev.md

```diff
@@ -0,0 +1,158 @@
+---
```

Comment:
```suggestion
---
eip: 7603
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.